addons/sms/static/src/components/message/message_tests.js

```diff
@@ -122,23 +124,22 @@ QUnit.test('Notification Sent', async function (assert) {
 QUnit.test('Notification Error', async function (assert) {
     assert.expect(8);
 
-    await this.start({
-        intercepts: {
-            do_action(ev) {
-                assert.step('do_action');
-                assert.strictEqual(
-                    ev.data.action,
-                    'sms.sms_resend_action',
-                    "action should be the one to resend sms"
-                );
-                assert.strictEqual(
-                    ev.data.options.additional_context.default_mail_message_id,
-                    10,
-                    "action should have correct message id"
-                );
-            },
-        },
+    const bus = new Bus();
+    bus.on('do-action', null, payload => {
```

Comment:
2nd parameter is the owner of event listener. This is in API of `.on()` of `web.Bus` and the motivation I think is technical and it's a way to avoid memory leaks and/or zombie function handling calls. `null` is used to not bother providing an owner... Not sure if it's clean enough, but at least I just followed how AAB adapted `intercepts` of `do_action`.